todo_app/data/item_service.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pymongo
+
+from todo_app.models.Card import Card
+
+class ItemService:
+    def __init__(self):
+        connection_string = os.getenv("CONNECTION_STRING")
+        database_name = os.getenv("DATABASE_NAME")
+        client = pymongo.MongoClient(connection_string)
+        self.db = client[database_name]
+        self.item_collection = self.db['todo-items']
+    
+    def get_items(self):
+        return [ Card.from_database(item) for item in self.item_collection.find()]
+
+    def add_item(self, title):
+        new_item = {
+            "title": title,
+            "status": "To Do"
+        }
+        self.item_collection.insert_one(new_item)
+
+    def start_item(self, id):
+        self.__update_item_status(id, "Doing")
+
+    def complete_item(self, id):
+        self.__update_item_status(id, "Done")
+
+    def delete_item(self, id):
+        self.item_collection.delete_one({ "_id": id })
+        
+    def __update_item_status(self, id, status):
+        self.item_collection.update_one({ "_id": id }, { "$set": { "status": status }})
```

Comment:
I don't think this _quite_ works; when I 'start' one of your tasks it doesn't move! The reason is that this update doesn't currently match anything because [Mongo makes its ids `ObjectId`s, not strings](https://pymongo.readthedocs.io/en/stable/api/bson/objectid.html). We need to convert our requested id back to such an object in order to get this query to match anything:
```suggestion
        self.item_collection.update_one({ "_id": ObjectId(id) }, { "$set": { "status": status }})
```
(also need `from bson import ObjectId`)

Note the same issue for deleting cards

Reply:
I did read about this, and fully expected to have to do as you have suggested - but I found running locally that it seemed to 'just-work' as is. And after deploying this branch to azure again it seems to be working there (for me at least)